src/internal/components/configurationPanels/GeoPushpinConfigurationPanel.tsx

```diff
@@ -6,7 +6,8 @@ import UnsupportedProperties from "../configurationControls/UnsupportedPropertie
 export default class GeoPushpinConfigurationPanel extends ConfigurationPanelContent {
     public componentDidMount() {
         this.props.pushData({
-            references: null,
+            properties: this.props.properties,
+            references: this.props.references,
             ignoreUndoRedo: true,
         });
```

Comment:
:)
`const { properties, pushData , references } = this.props;`

Reply:
updated